### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,29 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at
 https://github.com/denoland/deno_install
 
+### 1.0.1 / 2020.05.19
+
+- feat(std/node): fs.writeFileSync polyfill (#5414)
+- feat: add std/hash/fnv (#5403)
+- fix(std/hash): SHA1 hash of Uint8Array (#5086)
+- fix(std/node): fs.readFile should take string as option (#5316)
+- fix(std/testing): Provide message and diff for assertStrictEq (#5417)
```
Comment (nayeemrmn) [inline Releases.md:15]: If `std` isn't going to be bumped at the same time (?), these should be listed later under `### std/0.52.0`.

**Comment by ry** [comment]
holding release to investigate https://github.com/denoland/deno/issues/5614

File: Releases.md (hunk 1 above)
Comment (ry) [reply Releases.md:15]: we might as well bump std too

**Comment by ry** [comment]
holding for https://github.com/denoland/deno/issues/5658

**Comment by kt3k** [comment]
How about updating minor version (1.1.0 in this case) instead of patch version for periodical releases? ([Semver](https://semver.org/) seems suggesting that it should update minor version if it has any feature addition.)

**Comment by ry** [comment]
@kt3k there are no new features in the CLI - only in std - which is versioned differently now.

I've updated the release notes to make this more clear.

**Comment by kt3k** [comment]
@ry Thank you for the clarification! Now it's clear that the cli doesn't have a new feature with this release.

**Pull request merged**